config/features/config.go

```diff
@@ -182,6 +183,11 @@ func ConfigureBeaconChain(ctx *cli.Context) error {
 		logDisabled(disablePeerScorer)
 		cfg.EnablePeerScorer = false
 	}
+	cfg.DisableReorgLateBlocks = true
```

Comment:
Optional, I wish we have `EnableReorgLateBlocks ` instead of `DisableReorgLateBlocks`. The flag is `enable-reorg-late-blocks` and it makes everything a bit easier to read

Reply:
I want to make this default later, so the permanent config will be Disable, that's why I internally made this so that the changes are minimal when we flip the flag